node_defs.lua

```diff
@@ -327,70 +327,68 @@ for i in ipairs(moretrees.treelist) do
 			},
 		})
 
-		if moretrees.enable_stairs then
-			if minetest.get_modpath("moreblocks") then
+		if minetest.get_modpath("moreblocks") then
 
-	--			stairsplus:register_all(modname, subname, recipeitem, {fields})
+--			stairsplus:register_all(modname, subname, recipeitem, {fields})
```

Comment:
Comment can be removed IMO.

Reply:
https://github.com/mt-mods/moretrees/pull/41/commits/10b317d3f19c137a66cd31135a10ec4c1171f72d